docs/topics/kotlin-doc.md

```diff
@@ -1,15 +1,13 @@
-[//]: # (title: Document Kotlin code: KDoc and Dokka)
+[//]: # (title: Document Kotlin code: KDoc)
 
 The language used to document Kotlin code (the equivalent of Java's Javadoc) is called **KDoc**. In its essence, KDoc
 combines Javadoc's syntax for block tags (extended to support Kotlin's specific constructs) and Markdown for
 inline markup.
```

Comment:
Just thinking out loud: I wonder if we're past the point of needing to compare Kotlin with Java. It totally made sense in the beginning as people were primarily migrating from Java, but how many users/beginners are there now that are not familiar with Java and Javadoc?

Reply:
I actually asked this question recently and apparently the majority of users are still Java developers migrating to Kotlin. So I think for now we should keep it as it is. Perhaps if the audience type shifts in future we can reconsider.

Reply:
Thanks for sharing!